backup_migrate.install

```diff
@@ -505,3 +505,49 @@ function backup_migrate_update_1006() {
   }
 }
 
+/**
+ * NodeSquirel support has been removed.
+ */
+function backup_migrate_update_1007() {
+  backup_migrate_include('destinations');
```

Comment:
need to include `module_load_include('backup_migrate', 'backup_migrate', 'module');` first

Reply:
Thanks @jenlampton, I've added the line.